base/setup/lib/lang/ja-JP.h

```diff
@@ -2,7 +2,7 @@
 
 MUI_LAYOUTS jaJPLayouts[] =
 {
-//    { L"0411", L"e0010411" },
+//    { L"0411", L"e0010411" }, // We don't have preinstalled IMEs.
```

Comment:
```suggestion
//    { L"0411", L"E0010411" }, // We don't have preinstalled IMEs.
```
(to make it similar with the rest)